docs/sc-tools/sc_machine.md

```diff
@@ -4,18 +4,18 @@
     This is correct for only versions of sc-machine that >= 0.10.0.
 ---
 
-Sc-machine is a sc-memory with opportunity to add modules consistent with the general sc-machine interface. All such modules are named extensions, must be shared libraries (with file extension `.so` or `.dylib`) and stored in common directory (by default it is `./bin/extensions`). By default, sc-machine contains sc-server that allows to communicate with sc-memory through network. You can find `sc-machine` in a `bin` directory.
+Sc-machine is a sc-memory with opportunity to add modules consistent with the general sc-machine interface. All such modules are named extensions, must be shared libraries (with file extension `.so` or `.dylib`) and stored in common directory (by default it is `./build/<Release|Debug>/lib/extensions`). By default, sc-machine contains sc-server that allows to communicate with sc-memory through network. You can find `sc-machine` in a `/build/<Release|Debug>/bin` directory.
 
 ```sh
 Usage:
-  ./bin/sc-machine [options]
+  ./build/<Release|Debug>/bin/sc-machine [options]
 
 Required options:
   --storage|-s <directory>                Provide a path to directory with knowledge base binaries. Knowledge base binaries should be generated by sc-builder binary.
                                           This path can also be provided via the `storage` option in the [sc-memory] group of the configuration file specified with --config|-c.
                                           If both options are provided, the value from --storage|-s takes precedence.
   --config|-c <config-name>.ini           Provide a path to configuration file. If `storage` option is not provided as argument, then this file should contain `storage` option in `[sc-memory]` group.
```

Comment:
-c is not required anymore

Reply:
No, It's the same principle as sc-builder.